packages/client/hmi-client/src/components/model/model-parts/tera-model-error-message.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<aside>
+		<Message v-if="hasErrors" severity="error">
+			<ul>
+				<li v-for="item in errors" :key="item.id" @click.stop="filterByItem(item.id)">
+					{{ item.content }}
+				</li>
+			</ul>
+		</Message>
+		<Message v-if="hasWarnings" severity="warn">
+			<ul>
+				<li v-for="item in warnings" :key="item.id" @click.stop="filterByItem(item.id)">
+					{{ item.content }}
+				</li>
+			</ul>
+		</Message>
+	</aside>
+</template>
+
+<script setup lang="ts">
+import { ModelError, ModelErrorSeverity } from '@/model-representation/service';
+import { computed } from 'vue';
+import Message from 'primevue/message';
+
+const props = defineProps<{
+	modelErrors: ModelError[];
+}>();
+
+const emit = defineEmits(['filter-item']);
```

Comment:
```suggestion
const emit = defineEmits<{
  (e: 'filter-item', id: string): void;
 }>();
```
Could define like this to provide more detail as to the params the callback will receive